build.gradle

```diff
@@ -39,13 +42,12 @@ dependencies {
 	}
 	implementation configurations.localDeps
 
-	compileOnly "com.google.errorprone:javac:9+181-r4173-1"
-
+	errorprone("com.google.errorprone:error_prone_core:2.3.3")
+	errorproneJavac("com.google.errorprone:javac:${versions.errorproneJavacVersion}")
```

Comment:
These two lines use parentheses `()` around the argument, whereas the next two don't. Can these be more uniform?

Reply:
Thanks, fixed!